index.html

```diff
@@ -0,0 +1,121 @@
+            <circle id="Ellipse 4" cx="247" cy="563.096" r="10" fill="#6070FF"/>
+            <path id="Ellipse 13" d="M334.907 465.499C336.426 463.689 337.498 461.547 338.036 459.246C338.573 456.945 338.562 454.549 338.001 452.254C337.441 449.958 336.348 447.826 334.81 446.032C333.273 444.237 331.335 442.829 329.152 441.923C326.97 441.017 324.604 440.637 322.248 440.815C319.891 440.993 317.61 441.723 315.588 442.947C313.566 444.171 311.861 445.853 310.611 447.858C309.36 449.864 308.6 452.135 308.39 454.489L314.841 455.063C314.96 453.722 315.393 452.428 316.106 451.285C316.818 450.143 317.79 449.184 318.941 448.487C320.093 447.79 321.393 447.374 322.735 447.273C324.078 447.171 325.425 447.388 326.669 447.904C327.912 448.42 329.016 449.222 329.892 450.244C330.768 451.267 331.391 452.481 331.71 453.789C332.029 455.097 332.036 456.462 331.73 457.773C331.423 459.084 330.813 460.304 329.947 461.335L334.907 465.499Z" fill="#EBEBFF"/>
+            <path id="Ellipse 12" d="M318.497 429C318.278 432.194 319.337 435.344 321.441 437.757C323.545 440.17 326.521 441.649 329.715 441.867C332.909 442.086 336.059 441.027 338.472 438.923C340.885 436.82 342.364 433.843 342.583 430.649L318.497 429Z" fill="#6070FF"/>
+            <rect id="Rectangle 6" x="-34" y="37.4868" width="51.9898" height="9.30073" transform="rotate(-46.1415 -34 37.4868)" fill="#EBEBFF"/>
+            <path id="Ellipse 11" d="M-11.8095 24.6419C-13.8144 29.3377 -13.8718 34.6376 -11.969 39.3757C-10.0663 44.1137 -6.35924 47.9019 -1.66346 49.9068C3.03231 51.9117 8.3322 51.969 13.0703 50.0663C17.8084 48.1635 21.5965 44.4565 23.6014 39.7607L-11.8095 24.6419Z" fill="#6070FF"/>
+            <circle id="Ellipse 8" cx="221.5" cy="13.5" r="7" fill="white" stroke="#CBCDFF"/>
+            <circle id="Ellipse 14" cx="303.5" cy="216.5" r="5" fill="white" stroke="#CBCDFF"/>
+            <circle id="Ellipse 15" cx="60.5" cy="548.5" r="7" fill="white" stroke="#CBCDFF"/>
+            <path id="Ellipse 3" d="M421.241 74.9701C417.195 67.7876 411.514 61.659 404.659 57.0803C397.803 52.5017 389.966 49.6018 381.782 48.6154C373.597 47.6291 365.295 48.584 357.548 51.4028C349.802 54.2216 342.827 58.8252 337.191 64.8407C331.554 70.8562 327.413 78.1145 325.103 86.0281C322.793 93.9416 322.379 102.288 323.895 110.391C325.411 118.494 328.813 126.127 333.828 132.67C338.842 139.214 345.326 144.485 352.756 148.056L362.543 127.695C358.31 125.661 354.616 122.658 351.759 118.93C348.903 115.202 346.964 110.854 346.101 106.238C345.237 101.621 345.473 96.8662 346.789 92.3579C348.105 87.8496 350.464 83.7145 353.675 80.2874C356.886 76.8604 360.86 74.2378 365.273 72.6319C369.686 71.026 374.416 70.482 379.079 71.044C383.741 71.6059 388.206 73.258 392.112 75.8664C396.017 78.4748 399.254 81.9663 401.559 86.0582L421.241 74.9701Z" fill="#6070FF"/>
+            <rect id="Rectangle 4" x="294" y="61.79" width="95.5804" height="22.021" transform="rotate(-30 294 61.79)" fill="#EBEBFF"/>
+          </g>
+        </svg>
+      </div>
```

Comment:
- [ ]  As the previous reviewer suggested your social media icons should use ul tag to group them and li per icon. the Li tag should hold the icon as a link tag. This will order your HTML code in order and clean.  you can [refer ](https://www.w3schools.com/tags/tag_ul.asp)to this documentation 

- [ OPTIONAL] I recommend you to use icons from[ font awesome](https://fontawesome.com/start) to increase your hml code cleanness.